### PR TITLE
[launcher] Remove instance uid

### PR DESCRIPTION
**Pull request by MykolaSolyanko**

### CODE REVIEW

File: launcher/launcher.go [hunk 1]
```diff
@@ -787,6 +787,16 @@ nextNetInstance:
 			}
 		}
 
+		uid, err := launcher.storage.GetInstanceUID(netInstance)
```
Comment (al1img) [inline launcher/launcher.go:790]: Removing UID doesn't relate to `removeInstanceNetworkParameters`. IMHO it should not be here.
Also, tests failed on CI.

Comment (MykolaSolyanko) [reply launcher/launcher.go]: Done

File: launcher/launcher.go [hunk 2]
```diff
@@ -797,6 +804,43 @@ nextNetInstance:
 	}
 }
 
+func (launcher *Launcher) removeInstancesUID(instances []cloudprotocol.InstanceInfo) {
```
Comment (al1img) [inline launcher/launcher.go:807]: Looks like this function removes instances from DB as well. Should we rename it to `removeInstances`?

Comment (MykolaSolyanko) [reply launcher/launcher.go]: Agree, Done

**Review by al1img — APPROVED** [review]

**Pull request merged**